src/olympia/addons/tests/test_views.py

```diff
@@ -1590,6 +1590,11 @@ def _test_url(self):
     def _set_tested_url(self, param):
         raise NotImplementedError
 
+    def _make_unlisted(self):
```

Comment:
Should probably have made something like this a method of TestCase at the at start, heh.